types/site/_utils/add-pagination.d.ts

```diff
@@ -15,13 +15,11 @@
  */
 
 declare global {
-  export interface PaginatedPage<T = EleventyCollectionItem> {
-    date: Date;
-    description?: string;
-    href: string;
-    pagination: EleventyPagination<T>;
-    permalink: string;
-    title?: string;
+  export interface PaginatedPage<T extends {}> extends T {
+    elements: TODO[];
```

Comment:
That is intentional. The idea was that this could be a shared base for a VirtualCollectionItem as well as anything else that may get fed to the pagination function (tags). But there were annoying typing issues so I'm leaving it as is for now and will amend when tags gets updated to work like authors.

Also, the `PaginatableItem` doesn't work for this. Ill give it a more concrete typing but will need to ts-nocheck the tag-11tydata files.

Reply:
Let's go over this in our meeting